troposphere/views/guacamole.py

```diff
@@ -0,0 +1,87 @@
+import json
```

Comment:
There is a strong case to move this out of troposphere the python app and into atmosphere. Troposphere's python app is not doing much right now and it can potentially be swapped out for nodejs or something later on.